guide/11-performing-network-analyses/guide_to_network_analysis_part1.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>"However, there are situations <em>that </em><strong><em>where</em></strong> users prefer to use the customized NA services instead." replace 'that' with 'where'</p><p> </p><p>" to use the customized NA services on enterprise. " - Capitalize Enterprise as it refers to a product.</p><p> </p><p>When talking about class / module names, enclose them in <code>code</code> formatting - for instance - <code>network.analysis.find_routes</code></p><p> </p><p>"<strong>And</strong> the differences between APIs in these two modules are: " -&gt; remove 'And' and start sentence with The</p><p> </p><p>The last bullet point - "In situations where the output Feature Collection is oversized,..." is valid. But you should emphasize (in the previous bullet) that when using <code>features.user_proximity</code>, the inputs and outputs are Feature Service Layers, whereas in <code>network.analysis</code>, the inputs and outputs are in-memory <code>FeatureCollections</code>. Then the last point gets easier to understand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>